src/pathfora.js

```diff
@@ -88,7 +88,8 @@
         message: 'Message',
         company: 'Company',
         phone: 'Phone Number',
-        country: 'Country'
+        country: 'Country',
+        referrer: 'Referrer'
```

Comment:
should this be `referral` or even `referral-email`? i think technically in this case the referrer and the main email would be the same (e.g. mark hayden filled out the form and he referred sarah...which makes mark the referrer and sarah the referent....words are hard. just want it to be clear that the 1) its an email and 2) it doesn't belong to the person submitting the data.

also, i think we talked about doing some custom js for this upfront but we might be able to pull it off with just lql. need to run it by aaron to be sure but pretty sure we can just add another lql file that reads from default. if referral shows up we can add that user and ignore the uid and all that stuff....i believe.